go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/edgexfoundry/go-mod-core-contracts/v2
 go 1.19
 
 require (
-	github.com/eclipse/paho.mqtt.golang v1.4.2
+	github.com/edgexfoundry/go-mod-messaging/v2 v2.3.0
```

Comment:
move all the messaging client the go-mod-messaging to eliminate this dependency
there is a potential risk of this recursive depedency
go-mod-messaging should use this version of contract

Reply:
Moved.